api/api/specs/routers.yaml

```diff
@@ -423,12 +424,13 @@ components:
           type: "string"
           format: "date-time"
           readOnly: true
-        project:
+        project_id:
           type: "integer"
         environment_name:
           type: "string"
         endpoint:
           type: "string"
+          nullable: true
```

Comment:
this likely should be `readOnly`, instead of `nullable`, as it's not editable by users

Reply:
Yep, okay that makes sense; I was thinking of the case whereby routers get created but fail (and have no endpoint) or aren't deployed and `readOnly` still works perfectly in that case.